api/models/Survey.js

```diff
@@ -17,6 +17,8 @@ module.exports = class Survey extends Model {
                         models.Survey.hasMany(models.SurveyResult);
                         models.Survey.belongsToMany(models.Region, {through: 'surveyregion'});
                         models.Survey.belongsToMany(models.Office, {through: 'surveyoffice'});
+                        models.QuestionType.hasOne(models.Survey);
```

Comment:
ok my misunderstanding. 

each survey will still have a homogenous type of questions
- multiple choice 
- multiple choice with intensity
- intensity based questions

and these will be all scored differently